### DISCUSSION TITLE
Plans for 2024

### DISCUSSION THREAD

**Discussion by ghxstweb** [fastaddons/GroupSpeedDial#331] [General]
Hi @Juraj-Masiar and @fastaddons, do either you have any plans for the speed dial thisyear, any milestones or things to work on? I'm relatively experienced in javascript although not so much with the API for firefox integration, and would be interested in working on a project with you all if interested. Thanks in advance for your time!

**Comment by Juraj-Masiar** [comment]
Hello,
There is so many features planned, my todo list is only getting longer over the years. Even though I work on it every day full time.
For example, for the past several weeks I've been working on a new dynamic grid layout with custom size dials (resizing will be implemented today), look:

https://github.com/fastaddons/GroupSpeedDial/assets/6358971/e5b3f8ca-b84d-4e45-98c5-c09aa35a2137

However, this is not an open-source project, so I can't share the source with any 3rd party.
It's really nice of you that you want to help, but it's not that easy... :(
